src/components/RestaurantList/RestaurantList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ListWrapper } from "./RestaurantListElements";
+import { LIST_EXAMPLE } from "../../common/ListExample.jsx";
+
+export const RestaurantList = () => {
+  return (
```

Comment:
If the function has no logic in return, we can skip using return word and curly brackets. Like this:

```
export const RestaurantList = () => (
   <ListWrapper>
    ...
   </ListWrapper>
);
```

Reply:
I would paste fetching data in this component, so it won't be without logic. Probably in the same component will be logic with filtering

Reply:
+1